.vscode/settings.json

```diff
@@ -0,0 +1,19 @@
+{
+    "python.linting.pylintEnabled": false,
+    "python.linting.flake8Enabled": true,
+    "python.linting.mypyEnabled": true,
+    "python.linting.enabled": true,
+    "python.testing.pytestArgs": [
+        "--cov=python3_pip_skeleton",
```

Comment:
Should be cothread

Reply:
Resolved in https://github.com/dls-controls/cothread/pull/41/commits/c49161003423171e6b7f700dc5acb24c8d7fd9f7